### ISSUE TITLE
kokoro failures due to centos8 end of life

### ISSUE THREAD

**Issue by martijnvans** [GoogleCloudPlatform/ops-agent#373]
```
#100 0.702 + yum -y update
#100 1.150 CentOS Linux 8 - AppStream                      313  B/s |  38  B     00:00    
#100 1.154 Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist
```

https://www.centos.org/centos-linux-eol/

I guess we should delete centos8 from everywhere we can.

**Comment by martijnvans** [comment]
this is causing all kokoro builds to be failing across the board, and we have numerous PRs pending, so this is relatively urgent.

**Comment by igorpeshansky** [comment]
We should probably replace CentOS8 with Rocky Linux 8 instead (I've already tested that it works).

**Comment by igorpeshansky** [comment]
This seems to be fixed via #375, #376, and #377.
